pages/en/lb3/datasources.json.md

```diff
@@ -33,6 +33,19 @@ For example:
   }
 }
 ```
+To store data in a file for memory connector, add <code> "file": <i>"file_name"</i></code>
```

Comment:
> To make the memory connector store data in a file so it persists beyond application restarts, add ... 

Add period at end of sentence.

Add cross-ref:
For more information see [Memory connector - data persistence](Memory-connector.html#data-persistence).